(DOCSP-23900): Removing product source constants from app-services topics (#214)
mongodb/docs-app-services: source/reference/find-your-project-or-app-id.txt

````````````````````

Log into your `App Services <https://realm.mongodb.com?tck=docs_realm>`_

> console to view your App Services Applications. 

Reviewer: ```suggestion
console to view your App Services Apps. 
```

      applications you have created. There you can navigate to the dashboard
      of a specific app. To begin, log into
      `App Services <https://realm.mongodb.com?tck=docs_realm>`_ to view your

>       App Services Applications. Click your application to enter your

Reviewer: ```suggestion
      App Services Apps. Click your application to enter your
```

>       App Services Applications. Click your application to enter your
>       application dashboard.

Reviewer: ```suggestion
      App Services App. Click your App to enter its dashboard.
```
